### PR TITLE
Add a Swift Playground as an example

### PR DESCRIPTION
**Pull request by andrewheard**
Added a starter Swift Playground for generating text from a prompt.

### CODE REVIEW

File: Examples/GoogleAIPlayground.swiftpm/GenerateTextView.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      HStack {
+        TextField("Enter a prompt...", text: $prompt)
+        Button("Send", systemImage: "arrow.up.circle.fill") {
+          generateText()
+        }.labelStyle(.iconOnly)
+      }
+    }.padding()
+  }
+
+  func generateText() {
```
Comment (peterfriese) [inline Examples/GoogleAIPlayground.swiftpm/GenerateTextView.swift:29]: I think we should rename this to `send` or `sendMessage`, like in the other samples.

File: Examples/GoogleAIPlayground.swiftpm/GenerateTextView.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  @State var prompt: String = ""
+
+  var body: some View {
+    VStack {
+      ScrollView {
+        Text(text)
+      }
+      Spacer()
+      HStack {
+        TextField("Enter a prompt...", text: $prompt)
```
Comment (peterfriese) [inline Examples/GoogleAIPlayground.swiftpm/GenerateTextView.swift:21]: Add an `onSubmit` handler so people can just hit [enter] to send the message.

File: Examples/GoogleAIPlayground.swiftpm/GenerateTextView.swift [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        Button("Send", systemImage: "arrow.up.circle.fill") {
```
Comment (peterfriese) [inline Examples/GoogleAIPlayground.swiftpm/GenerateTextView.swift:22]: Use a `Button` with a nested `Image`, like in the other samples

**Review by peterfriese — CHANGES_REQUESTED** [review]
Great idea, thanks for putting this together!

I found it pretty difficult to edit the text in the Swift Playground app, as the code is formatted using 2 spaces, but the Playground app doesn't seem to allow changing the indentation width. So you constantly end up fighting the editor, which is a pretty bad user experience.

We should change the formatter configuration for this example.

**Comment by andrewheard** [comment]
Thanks for the review, @peterfriese! Closing this in favour of adding docs instead (#72).

**Pull request closed without merging**